Questão 1: Considerando o programa abaixo e sabendo que o usuário digitou 3 e 6 (nessa ordem), preencha no campo resposta o valor final da variável total:

In [1]:
num1 = int(input("Digite um número"))
num2 = int(input("Digite outro número"))

total = num1 + num2/3

print("O novo número é ", total)

Digite um número3
Digite outro número6
O novo número é,  5.0


Questão 2: Considerando o programa abaixo e sabendo que o usuário digitou 200 e 40 (nessa ordem), o que seria impresso na ordem exata de execução?

In [2]:
salario = float(input("Digite o valor do salário"))
aumento = float(input("Digite o percentual de aumento"))

novoSalario = salario*(1+aumento/100)

print(novoSalario)
print(salario)

Digite o valor do salário200
Digite o percentual de aumento40
280.0
200.0


Questão 3: O programa abaixo foi executado diversas vezes e, após o usuário digitar um valor, o programa imprimiu a situação do aluno. Para cada valor digitado, coloque do lado a situação do aluno (“Reprovado”, “Recuperação” ou “Aprovado”):

In [8]:
media = float(input("Qual a média do aluno?"))

if 0 < media <=5:
    print("Reprovado")
else:
    if 5 < media < 6:
        print("Recuperação")
    else:
        print("Aprovado")

Qual a média do aluno?4
Reprovado


| Valor Digitado | Situação |
| --- |--- |
| 5 | Reprovado |
| 8 | Aprovado |
| 1 | Reprovado |
| -5.5 | Aprovado |
| 4 | Reprovado |

Questão 4: Para cada execução do programa abaixo, indique na tabela os valores finais das variáveis precoA e precoB:

In [13]:
precoA = int(input("Digite o valor do produto 1"))
precoB = int(input("Digite o valor do produto 2"))

if precoA>precoB:
    precoB = precoB*2
else:
    if precoA==precoB:
        precoB = precoB*1.5
    else:
        precoA = precoB*2 + 10

print(precoA)
print(precoB)

Digite o valor do produto 140
Digite o valor do produto 250
110
50


| Valor Digitado 1 | Valor Digitado 2 | precoA | precoB
| --- |--- | --- |--- |
| 100 | 200 |410 | 200 |
| 50 | 20 |50 | 40 |
| 60 | 60 |60 | 90 |
| 70 | 20 |70 | 40 |
| 40 | 50 |110 | 50|

Questão 5: Ao executar o programa abaixo, qual é a média final do aluno?

In [14]:
notas = (10,6,5,9)
soma = 10
for nota in notas:
    soma = soma + nota
print("Média", soma/4)

Média 10.0


Questão 6: Ao executar o programa abaixo, o usuário digitou DF e SP (nessa ordem). Qual será o valor final da variável percentual?

In [15]:
origem = input("Digite um estado")
destino = input("Digite outro estado")

percentual = 18

if origem=="SP" and destino=="DF":
    percentual = 12
if origem=="GO" or destino=="DF":
    percentual = 7
    
print("Percentual", percentual)

Digite um estadoDF
Digite outro estadoSP
Percentual 18


Questão 7: Um usuário pretendia digitar os números 2, 3, 2, 1, 4, -3, 1, 4, 1, 2, 0, mas o programa encerrou antes que ele digitasse todos os valores. Qual foi o último valor digitado?

In [20]:
soma = 0
rodadas = 10
num = 0
while soma <= rodadas:
    num = int(input("Digite um número de 1 a 10"))
    soma = soma + num
    if soma > 20:
        break
print(soma/4)

Digite um número de 1 a 102
Digite um número de 1 a 103
Digite um número de 1 a 102
Digite um número de 1 a 101
Digite um número de 1 a 104
3.0


Questão 8: Ao executar o programa abaixo, qual será o valor final da variável mediaFinal:

In [18]:
notas = (10,5,4,5)
mediaFinal = 0
soma = 0

for i in notas:
    if i < 4:
        mediaFinal = 0
    else:
        soma = soma + i

mediaFinal = soma / 4
print(mediaFinal)

6.0


Questão 9: Uma faculdade deseja implementar um programa para cálculo de médias de notas de provas, de modo que seja calculada a média independente de quantas notas o aluno terá. Ou seja, o programa deve calcular a média para uma turma com 3 provas e para outra turma com 4 provas. Continue a implementação abaixo desse programa considerando os requisitos acima, sabendo que você só precisa implementar de 3 a 4 linhas para o cálculo e mais uma para imprimir a média.

In [19]:
notas = (10,6,4,8,2,6)
quantidade = len(notas)
soma = 0

for nota in notas:
    soma += nota
print(soma/quantidade)

6.0
